src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+import { colors } from "../../global/colors";
+
+export const BtnWrapper = styled.a`
+    height: 20px;
+    border-radius: 4px;
+    text-transform: uppercase;
+    background-color: ${colors.primary};
+    border: none;
+    padding: 6px 12px 6px 12px;
+    margin: 2px 7px;
+    color: ${colors.bgColor};
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 14px;
+    transition-duration: 0.4s;
+    cursor: pointer;
+    float: right;
```

Comment:
It is better not to use float, use flexbox or grid instead

Reply:
Got rid of float everywhere